README.md

```diff
@@ -21,7 +21,11 @@
 
 # 📖 [AWESOME BOOKS ES6] <a name="about-project"></a>
 
+<<<<<<< HEAD
+**Awesome-books-ES6** is a project that we are applying our knowledge on HTML and CSS and Javascript to showcase our most crisp and pristine projects.
+=======
```

Comment:
- [ ] I'm noticing some random texts on your readme like the "<<<<HEAD" text above please make sure to keep your readme clean and professional at all times as this is the first thing people are going to look at when they visit your project. ✔
- [ ] This also applies to other places on the readme where you can see the text "<<<<HEAD".